website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,26 @@
+The File Input component uses the native HTML input with `type="file"` under the hood but adds visual styling to match other HDS components.
+
+
+## Truncation
+
+Truncation is built into the native component and uses the width of the container to determine when the text gets truncated. It is not based on a character limit. 
+
+!!! Info
+
+Due to limitations in Figma, truncation will occur at the end of the text, while in the browser, truncation occurs in the middle. 
+
+![visual difference in truncation between figma and browser](/assets/components/form/file-input/file-input-truncation.png)
+!!!
+
+## Multiple files
+
+Multiple files can be added at once when the `multiple` attribute is enabled in code. When more than one file is selected, the text will change to show the number of files you’ve selected, e.g., “2 files”. *The exact language used may differ between browsers.*
+
+!!! Info
+
+*A note about browser inconsistency*
+
+Because the File Input is based on the native HTML input, the content and visual appearance may vary slightly across browsers.
+
+![Examples of file input states rendered in different web browsers](/assets/components/form/file-input/file-input-browsers.png)
+!!!
```

Comment:
Including this here below "##Multiple files" still makes it seem to me like it's supposed to be related except it's not. Could we maybe move it more towards the top under this text: "The File Input component uses the native HTML input with type="file" under the hood but adds visual styling to match other HDS components."?

@heatherlarsen @jorytindall what do you think?

Reply:
It seems like the info block is doing a lot of heavy lifting here. Agreed, moving it under the intro sentence and potentially adding a headline for "Browser inconsistency" or similar (and not wrapping it in the info block.

Reply:
Adding a headline and removing the Info block makes sense to me too.

Reply:
@andgen404 @jorytindall 

I applied Andrew's change to move the content and then added a headline, what do you think?

<img width="866" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/4e2bf806-8320-470f-b895-0307b7340c45">

Reply:
Thank you @KristinLBradley I think this change looks good to me 🙂